examples/readme.md

```diff
@@ -0,0 +1,25 @@
+# `dattri` examples
+This folder contains bite-sized examples which can help users to build their own application by `dattri`.
```

Comment:
This folder contains bite-sized examples that can help users build their own applications with `dattri`.